src/benchmarklib/tpch/tpch_benchmark_item_runner.cpp

```diff
@@ -489,4 +489,8 @@ std::string TPCHBenchmarkItemRunner::_substitute_placeholders(const BenchmarkIte
   }
 }
 
+void TPCHBenchmarkItemRunner::load_dedicated_expected_results() {
+  // do nothing; dedicated expected result verification is not supported for the TPCHBenchmarkItemRunner
```

Comment:
If it is not necessary to implement this method, simply don't make it pure virtual.